pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -168,29 +221,41 @@ Function | Description
 
 For any function you defined as an `export` in your `gulpfile.js`, you can run `npx gulp [function]`.
 
-## Step 5: Initialize your project
-Initialize your project to copy all the necessary image, font, and Javascript assets from the source code.
+For example, if you wanted to have the ability to tell Gulp to compile your Sass, you'd need to first export the `compile` task for your project by adding this line to your `gulpfile.js`:
+
+```
+exports.compile = uswds.compile;
+```
+
+Once exported, you can compile your code by running the following in the command line from your project root:
+
+```
+npx gulp compile
+```
+
+### Step 7: Initialize your project
+
+When your `gulfile.js` is ready, initialize your project to copy all the necessary Sass, image, font, and Javascript assets from the USWDS source code.
```

Comment:
Missed the typo here in `gulpfile`.

Reply:
Thanks. Fixed [da0713f](https://github.com/uswds/uswds-site/pull/1766/commits/da0713f8b2fdb0e70227855f62710eedc4d23270)